services/brig/test/resources/nexmo.txt

```diff
@@ -0,0 +1,2 @@
+key: "dummy"
```

Comment:
is this file `yaml`? maybe change the extension from `.txt` to `.yaml`?

Reply:
👍